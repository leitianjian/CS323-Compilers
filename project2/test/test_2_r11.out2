Error type 11 at Line 0: applying function invocation operator (foo(...)) on non-function names
Error type 8 at Line 16: function return type is not matched
