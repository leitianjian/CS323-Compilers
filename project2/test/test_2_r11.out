Error type 11 at line 16: applying function invocation operator on non-function names
